README.md

```
@@ -1,26 +1,152 @@
-npm run dev
-# or
+# Start development server
 pnpm dev
-# or
-yarn dev
+
+# Build for production
+pnpm build
 ```
 
 Open http://localhost:3000 with your browser to see the result.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)